mesonbuild/build.py

```diff
@@ -831,14 +831,17 @@ def add_deps(self, deps):
                 for l in dep.libraries:
                     self.link(l)
                 # Those parts that are external.
-                extpart = dependencies.InternalDependency('undefined',
-                                                          [],
+                extpart = dependencies.DeclaredExternalDependency('undefined',
                                                           dep.compile_args,
                                                           dep.link_args,
-                                                          [], [], [])
+                                                          [])
```

Comment:
__[Flake8]__

[E128] continuation line under-indented for visual indent


<!-- Comment by [SideCI](https://sideci.com) -->